app/client/components/accountoverview/accountOverviewCard.tsx

```diff
@@ -34,6 +34,9 @@ interface AccountOverviewCardProps {
 
 export const AccountOverviewCard = (props: AccountOverviewCardProps) => {
 	const mainPlan = getMainPlan(props.productDetail.subscription);
+	if (!mainPlan) {
+		throw 'mainPlan does not exist in accountOverviewCard';
```

Comment:
The `<ErrorBoundary>` component is expecting an error object so you'll need to wrap the string in `new Error()` otherwise it gets logged as `undefined: undefined`.  (Internally it's returning `${error.name}: ${error.message}`.) I've pulled down the branch and double-checked as I wasn't sure if that was the case 😅 

```suggestion
		throw new Error('mainPlan does not exist in accountOverviewCard');
```